vue-components/src/components/Lookup.vue

```diff
@@ -111,42 +122,85 @@ export default Vue.extend( {
 	},
 
 	methods: {
+		canShowMenu( currentSearchInput: string ): boolean {
+			return currentSearchInput.length > 0;
+		},
 		onInput( value: string ): void {
-			this.hasItemSelected = false;
+			this.enterKeyPressed = false;
+			this.selectedItemIndex = -1;
+			this.showMenu = this.canShowMenu( value );
 
 			// the following comment generates the event's description for the docs tab in storybook
 			/**
 			 * Enables the `searchInput` prop to be used with the `.sync` modifier. It's used to transport the value of
 			 * the Lookup component's inner `<input>` element to the parent component.
 			 */
 			this.$emit( 'update:searchInput', value );
+			this.$emit( 'input', null );
 		},
 
 		onSelect( menuItem: MenuItem ): void {
-			this.hasItemSelected = true;
+			this.hasActiveElement = true;
```

Comment:
While looking at the component together with @SaiSan-WMDE we noticed that once the selection is made, and we navigate away from the component, this `hasActiveElement` does not revert back to `false` which causes the items of the menu to appear active when hovered.

According to @SaiSan-WMDE The active style should only appear when the mouse button is down.